### PR TITLE
Portenta X8: Global Post Update

### PR DESCRIPTION
**Pull request by TaddyHC**
## What This PR Changes
It fixes, changes, and updates the following content:
- User Manual
- Multi-Protocol Gateway
- Waves Fleet Management

Edit: Additional tutorials have received updates regarding content.

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by martab1994 — APPROVED** [review]
Looks great to me =)

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Review WIP

User manual changes requested related with permalinks

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 1]
```diff
@@ -76,57 +80,59 @@ The full pinout is available and downloadable as PDF from the link below:
 
 ### Datasheet
 
-The full datasheet is available and downloadable as PDF from the link below: 
-* [Portenta X8 Datasheet](https://docs.arduino.cc/static/77d2be9902708fcb73b19d0f1173c6bb/ABX00049-datasheet.pdf)
+The full datasheet is available and downloadable as PDF from the link below:
+* [Portenta X8 Datasheet](https://docs.arduino.cc/static/e12188dbbde2deba83f37965615c3c5a/ABX00049-datasheet.pdf)
```
[the remark this answers is not in the document]
Comment (sebromero) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:84]: > It is clear to me we are doing here something wrong
> 
> The hash is changing over time which means we should use a static link that will redirect to the last version of these resources.
> 
> @jhansson-ard or @sebromero do you know what is the right format to link to the datasheets?
> 
> Should be something similar to:
> 
> https://docs.arduino.cc/resources/datasheets/AFX00001-AFX00002-AFX00003-datasheet.pdf
> 
> Which means:
> 
> https://docs.arduino.cc/resources/datasheets/ABX00049-datasheet.pdf
@jcarolinares 
Correct. It copies those resources to the /resources folder which makes them available under the said URL. Here's where that happens: https://github.com/arduino/docs-content/blob/main/.github/workflows/deploy-prd.yml#L28-L34

[the remark this answers is not in the document]
Comment (jhansson-ard) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:84]: @jcarolinares and yes, that is the correct way of adding the datasheet 🙏🏼

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 2]
```diff
@@ -76,57 +80,59 @@ The full pinout is available and downloadable as PDF from the link below:
...
 
 ### Schematics
 
-The full schematics are available and downloadable as PDF from the link below: 
+The full schematics are available and downloadable as PDF from the link below:
 * [Portenta X8 Schematics](https://docs.arduino.cc/static/c92dbfc6c6a7c3f79c9987d36f017e48/ABX00049-schematics.pdf)
```
[the remark this answers is not in the document]
Comment (jhansson-ard) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:89]: @jcarolinares please update the word `datasheets` to `schematics` in the link here ⭐

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 3]
```diff
@@ -76,57 +80,59 @@ The full pinout is available and downloadable as PDF from the link below:
...
 
 ### STEP Files
 
-The full _STEP_ files are available and downloadable from the link below: 
+The full _STEP_ files are available and downloadable from the link below:
 * [Portenta X8 STEP files](https://docs.arduino.cc/static/5d95e348688a9678e04d6f4d0b994844/ABX00049-step.zip)
```
[the remark this answers is not in the document]
Comment (jhansson-ard) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:94]: I think we only have the special links for datasheets, schematics and pinouts ⭐

[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:94]: then keep this one as it is @TaddyHC !

[the remark this answers is not in the document]
Comment (TaddyHC) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:94]: Alright, will do that, thanks!

**Review by jcarolinares — APPROVED** [review]
Looks good to me

**Pull request merged**